data/license.html

```diff
@@ -0,0 +1,1292 @@
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this file,
+   - You can obtain one at http://mozilla.org/MPL/2.0/.  -->
+
+<html lang="en">
+  <head>
+    <meta http-equiv="Content-Type" content="text/html;charset=utf-8">
+    <title>Valence licensing notices</title>
+
+    <style type="text/css">
+      .path {
+        font-family: monospace;
+      }
+
+      dt {
+        font-weight: bold;
+      }
+    </style>
+    <link rel="stylesheet" href="chrome://global/skin/about.css" type="text/css">
+  </head>
+
+  <body id="lic-info" class="aboutPageWideContainer">
+    <h1><a id="top"></a>Valence licensing notices</h1>
+
+    <div>
+
+    <p>Valence provides Remote Debugging Protocol Adapters for Firefox
+       Developer Tools. User documentation for Valence is available on
+       <a href="https://developer.mozilla.org/docs/Tools/Valence">MDN</a>.
+    </p>
+
+    <p>All of the <b>source code</b> to this product is
+       available under licenses which are both
+       <a href="http://www.gnu.org/philosophy/free-sw.html">free</a> and
+       <a href="http://www.opensource.org/docs/definition.php">open source</a>.
+       Instructions for building the binary components included in this product are also
+       <a href="resource://fxdevtools-adapters-at-mozilla-dot-org/data/building.html">
+       provided</a>.
+    </p>
```

Comment:
If you change the text to "instructions for obtaining the source for and building the binary components..." then that's fine. But otherwise, I think it's a bit obscure.